### ISSUE TITLE
Bug report: JSON to CSV

### ISSUE THREAD

**Issue by Neo23x0** [gchq/CyberChef#527]
### Summary

A valid JSON file cannot be converted to CSV. The reason is unknown. 

```json
{
    "cmd": "/usr/libexec/routined LAUNCHED_BY_LAUNCHD",
    "hostip": "192.168.1.162",
    "hostname": "prometheus.local",
    "level": "Info",
    "message": "Checking process",
    "module": "ProcessCheck",
    "name": "routined",
    "pid": 362,
    "scanner": "SPARK",
    "timestamp": "2019-03-29T13:28:16Z",
    "user": "neo"
}
```

### Example

https://gchq.github.io/CyberChef/#recipe=JSON_Beautify('%20%20%20%20',false)JSON_to_CSV(',','%5C%5Cr%5C%5Cn')&input=eyJjbWQiOiIvdXNyL2xpYmV4ZWMvcm91dGluZWQgTEFVTkNIRURfQllfTEFVTkNIRCIsImhvc3RpcCI6IjE5Mi4xNjguMTQuMTYyIiwiaG9zdG5hbWUiOiJwcm9tZXRoZXVzLmxvY2FsIiwibGV2ZWwiOiJJbmZvIiwibWVzc2FnZSI6IkNoZWNraW5nIHByb2Nlc3MiLCJtb2R1bGUiOiJQcm9jZXNzQ2hlY2siLCJuYW1lIjoicm91dGluZWQiLCJwaWQiOjM2Miwic2Nhbm5lciI6IlNQQVJLIiwidGltZXN0YW1wIjoiMjAxOS0wMy0yOVQxMzoyODoxNloiLCJ1c2VyIjoibmVvIn0K

**Comment by st0le** [comment]
It appears the Operation breaks on any datatype other than string. Your example works if I modify the input 
```

[{
    "cmd": "/usr/libexec/routined LAUNCHED_BY_LAUNCHD",
    "hostip": "192.168.1.162",
    "hostname": "prometheus.local",
    "level": "Info",
    "message": "Checking process",
    "module": "ProcessCheck",
    "name": "routined",
    "pid": "362",
    "scanner": "SPARK",
    "timestamp": "2019-03-29T13:28:16Z",
    "user": "neo"
}]
```

**Comment by n1474335** [comment]
Fixed in #547
